apps/web/modules/auth/login-view.tsx

```diff
@@ -256,12 +256,10 @@ PageProps & WithNonceProps<{}>) {
                     CustomStartIcon={<GoogleIcon />}
                     onClick={async (e) => {
                       e.preventDefault();
-                      const res = await signIn("google", {
+                      setLastUsed("google");
+                      await signIn("google", {
                         callbackUrl,
                       });
-                      if (res && !res.error) {
-                        setLastUsed("google");
-                      }
```

Comment:
Isn't it better to add provider to lastUsed when sign up is completed successfully?
Not sure why we are changing it?

Reply:
The code never reaches that line. For Oauth it is redirect. So the last line is signIn()

Reply:
Ahh makes sense 👍🙏